Source/Scene/CameraEventAggregator.js

```diff
@@ -262,7 +262,8 @@ define([
         }
         //>>includeEnd('debug');
 
-        this._eventHandler = new ScreenSpaceEventHandler(canvas);
+        this._eventHandler = new ScreenSpaceEventHandler(canvas, false);
```

Comment:
Nope. It would get triggered if we pass in `true` which is the default. I changed this class to use the new behavior.